### ISSUE TITLE
Initial QA for new search bar

### ISSUE THREAD

**Issue by jonallured** [artsy/reaction#2043]
- [x] [Fix positioning in force](https://github.com/artsy/force/pull/3590) - after #2042 gets merged
- [ ] Style for medium/small breakpoints (after fixing positioning ^^^, and after feedback from Jeffrey)
- [ ] The "return" icon is missing on active suggestion
    * [There is an issue for palette](https://github.com/artsy/palette/issues/283) to add this icon (and many others). As I noted in that issue, none of the icons have semantic `<title>` tags, so they may not be ready for creation yet.
- [x] Decorate input with `form` so that non-js searches will submit correctly
- [ ] Cross-browser testing!
- [ ] Verify keyboard usability
- [ ] Maybe set a permanent height of the box, so it doesn't dance around?
- [x] Maybe set explicit width/height of preview grid item image containers so that they don't dance around when images are loading?
- [ ] The loading spinner seems off-center in the preview pane.
- [ ] If I click on the items in the previews, the search box closes but I don't go anywhere!
---

Notes:

* When choosing a suggestion, the experience seems a little strange. The term is removed from the input and the suggestions disappear before the URL changes or the new page is loaded.
* Investigate what happens when you move breakpoints - it appears that there are actually two searches ATM!
    * Sounds like Jeffery will explore this more
* The SearchBar seems to snap in about a second after the page shows up

**Comment by jonallured** [comment]
Oops, closing in favor of the list over on #2042 😝

**Comment by pepopowitz** [comment]
We decided to extract a bunch of stuff from #2042, so I'm reopening & updating the list of remaining issues!

**Comment by pepopowitz** [comment]
Closing again because we moved these all to JIRA.
